MIP/mip-n/README.md

```diff
@@ -0,0 +1,148 @@
+# MIP-0: Release Conventions for Movement Technologies
+- **Description**: Release Conventions for Movement Technologies address the delivery of canonical artifacts to the Movement community.
+- **Authors**: [Liam Monninger](mailto:liam@movementlabs.xyz)
+- **Desiderata**: [MD-n](../MD/md-n)
+
+## Abstract
+
+Release Conventions for Movement Technologies standardize the process of delivering artifacts to the Movement community. By providing a coherent and repeatable process for releasing artifacts, these conventions ensure that Movement technologies are accessible and easy to use.
+
+The release conventions proposed herein are based on the usage of GitHub for source control and the tagging of GitHub releases. Guidelines for implementing programmatic access to release artifacts are also described. Finally, the intended relationship between releases and governance is outlined.
+
+## Motivation
+
+Movement technologies are continually evolving, and there's a need to ensure that the process of releasing artifacts is both organized and standardized. By establishing Release Conventions for Movement Technologies, we aim to facilitate the delivery of canonical artifacts to the Movement community. This will ensure that Movement technologies are accessible, easy to use, and well-vetted.
+
+## Specification
+
+The definitions of "artifact" and "release" are accepted from [MD-n](../MD/md-n) and submitted for formal adoption in [MG-n](../MG/mg-n) and [MG-k](../MG/mg-k). 
+
+We break the specification into four sections:
+
+1. **Standardized Release Naming Convention**: the section specifying the naming convention to be used for releases.
+2. **Standardized Source Control**: the section specifying the process for initiating releases from source control.
+3. **Programmatic Access to Release Artifacts**: the section specifying how release artifacts can be fetched programmatically and how software should be designed to this end.
+4. **Standard Relationship between Releases and Governance**: the section specifying the relationship between releases and governance as should be assumed the default. 
+
+### Standardized Release Naming Convention
+We propose the following naming convention for releases, inspired by the [Aptos release standard](https://github.com/aptos-labs/aptos-core/blob/main/RELEASE.md):
+
+```
+<software-unit>-v<major>.<minor>.<patch>-<release-type>
+```
+
+Where:
+- `<software-unit>`: the name of the software unit being released, e.g., the Suzuka Full Node. 
+- `<major>`, `<minor>`, `<patch>`: the version numbers of the release conforming to [Semantic Versioning](https://semver.org/).
+- `<release-type>`: the type of release, e.g., `stable`, `beta`, or `rc`.
```

Comment:
I would omit the suffix in the case of `stable`, as is the standard practice in SemVer.